crates/iron-remote-desktop/src/session.rs

```diff
@@ -0,0 +1,81 @@
+use wasm_bindgen::JsValue;
+use web_sys::{js_sys, HtmlCanvasElement};
+
+use crate::clipboard::ClipboardTransaction;
+use crate::error::IronError;
+use crate::input::InputTransaction;
+use crate::DesktopSize;
+
+pub trait SessionBuilder {
+    type Session: Session;
+    type IronError: IronError;
```

Comment:
suggestion: It’s probably fine to call this associated type `Error`.
```suggestion
    type Error: IronError;
```

Reply:
Done